src/bfx_tools/samtools.ml

```diff
@@ -150,6 +154,7 @@ let sort_bam_no_check ~(run_with:Machine.t) ?(processors=1) ~by input_bam =
     | `Read_name -> "sort" :: "-n" :: command
   in
   do_on_bam ~run_with input_bam ~product ~make_command
+    ~more_requirements:[`Memory `Big]
```

Comment:
Yea, my thinking was since it's hard to tell, making it an explicit option might easier. `Big` to this might be small or not even possible on other machines. Where is Big translated to a number?